### ISSUE TITLE
[feature]: support of passing arbitary amount to a taproot address 

### ISSUE THREAD

**Issue by vanditshah99** [lightninglabs/taproot-assets#755]
It will be a great help to have that feature to send an arbitarty amount to taproot address, So allowing us to create an address without specifying an amount or zero amount and let the asset send have the amount field to accept any amount for that taproot address

**Comment by Roasbeef** [comment]
I think this is possible, but fwiw that way the addresses were designed was that they're intended to be more or less single use, and also bind the amount into the address similar to the way things work on LN. 

For a receiver to detect a credit on-chain, they need to be able to derive the TAP leaf that's committed to in the normal taproot output. This leaf includes the amount requested. So if we want to keep that same flow (receives can be detected on-chain), then we'd need to lift that into the proof that's transmitted tot he receiver to conclude the transfer process.

**Comment by jayneel007** [comment]
@Roasbeef
The ability to create a 0-amount invoice or "Pay any amount" invoice is a very important feature for us. Many of our customers use our payment link solution to accept tips, donations, or crowdfunding, allowing the end user to decide the payment amount. This way, merchants can create a payment link for any amount, and we can generate an invoice with a 0 amount, enabling them to accept any payment amount on that invoice.

Is it possible to prioritize this feature?

**Comment by Roasbeef** [comment]
@jayneel007 yes, we'll address this while creating group key addresses: https://github.com/lightninglabs/taproot-assets/issues/291.

**Comment by levmi** [comment]
Latest Group Key Address design thoughts are in: https://github.com/lightninglabs/taproot-assets/issues/874
